### ISSUE TITLE
DOC: `fromnumeric.py`  - list of functions that are missing examples – tracking issue

### ISSUE THREAD

**Issue by InessaPawson** [numpy/numpy#22266]
Find below the functions from `fromnumeric.py` that are missing examples in the docstrings. If you have a suggestion for an example, please submit a pull request. 
To let everyone know that there is work in progress, link your PR to this issue. (If needed, refer to the [GitHub documentation](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue) on how to do it.)
   
- [x]     alltrue  (alias)
- [x]     copyto
- [x]     diag_indices_from
- [ ]     ~get_array_wrap~ (deprecated in https://github.com/numpy/numpy/pull/24154)
- [ ]     get_include
- [ ]     get_printoptions
- [ ]     ~getbufsize~  (requires experimenting and understanding its effect)
- [x]     msort  (technically an alias)
- [x]     product
- [x]     recfromcsv
- [x]     recfromtxt
- [ ]     ~round_  (alias)~
- [ ]     ~setbufsize~  (see getbufsize)
- [ ]     sometrue  (alias)
- [x]     tril_indices_from
- [x]     triu_indices_from

**IMPORTANT:** Don't close this issue or remove the *"sprintable"* label until all the PRs with examples are merged.

**Comment by simranmakandar** [comment]
I am giving this a shot as part of OSD @ GHC 2022

**Comment by sanjanamm98** [comment]
Looking into **product** function as part of the Grace Hopper OSD NumPy sprint

**Comment by dg3192** [comment]
I am giving this a shot as part of OSD @ GHC 2022

**Comment by pavani34** [comment]
I am giving this a try as part of OSD @ GHC 2022

**Comment by shreya-singh-tech** [comment]
Looking into this as part of the Grace Hopper OSD NumPy sprint

**Comment by MohitKumar020291** [comment]
Opened [PR](https://github.com/numpy/numpy/pull/24002). Please review it.

**Comment by MohitKumar020291** [comment]
Opened a new [PR](https://github.com/numpy/numpy/pull/24010). Please review it.

**Comment by bhavyaalekhya** [comment]
Hi! I'd like to work on this as a part of GHC OSD'23

**Comment by luxedo** [comment]
I worked on the `main` branch but it's already in version 2.1.

I created examples for `get_printoptions` https://github.com/numpy/numpy/pull/26488 and `get_include` https://github.com/numpy/numpy/pull/26489. `sometrue` is now on `ma/core.py` and it's an alias for `logical_or.reduce`.

I feel that this issue is outdated. If you want I can bring the changes to `maintenance/1.26.x` and add the `sometrue` documentation as well.
